src/api/app/views/webui/comment/_links.html.erb

```diff
@@ -6,11 +6,12 @@
     </a>
    </li>
   <% end %>
-  <% if comment.check_delete_permissions %>
+  <% if policy(comment).destroy? %>
       <li>
         <%= link_to(sprited_text('comment_delete', 'Delete'),
-                    {controller: :comments, action: :destroy,
-                     id: comment.id}, method: :delete) %>
+                    {controller: :comments, action: :destroy, id: comment.id},
```

Comment:
Because you can delete a comment accidentally, so I added a confirmation to prevent this kind of situation. Furthermore, in most of the cases, I think that a delete action needs a double check.